### PR TITLE
[docs] Merge both the components section

### PR DESCRIPTION
**Pull request by vchrombie**
This PR merges old and new components sections to create a dropdown Components section.

```
|____components
| |____scenarios.md
| |____workflow.md
| |____components.md
```
The images are added accordingly.

### CODE REVIEW

File: docs/components/scenarios.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+---
+layout: default
+title: Projects.json
+nav_order: 1
+parent: Configure GrimoireLab
+grand_parent: Components
+---
+
```
Comment (vchrombie) [inline docs/components/scenarios.md:8]: Please let me know if I need to update the front matter to create the dropdown as mentioned in the PR description.

File: docs/components/workflow.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## Components
+# Workflow
```
Comment (vchrombie) [inline docs/components/workflow.md:1]: same as above

**Review by vchrombie — COMMENTED** [review]
@vsevagen, could you please review the PR when you have time?

File: docs/components/scenarios.md (hunk 1 above)
Comment (VSevagen) [reply docs/components/scenarios.md:8]: In order to have the dropdown as per the PR description, we'll need one `index.md` file titled `Components` to act as the parent section. It needs to have the following config.
```
layout: default
title: Components
nav_order: 2
permalink: /components
has_children: true
has_toc: true
```

Then, the three dropdown element (`component.md`, `scenario.md` and `workflow.md`) should all the have following in the beginning of their respective md file,

```
---
layout: default
title: <replace with proper title, e.g Workflow>
nav_order: <replace with order number of this file in dropdown>
parent: Components
---
```

File: docs/components/workflow.md (hunk 2 above)
Comment (VSevagen) [reply docs/components/workflow.md:1]: It needs to have some config. See my comment above.

**Comment by VSevagen** [comment]
Also, if we can keep the content within 90 characters per line, it would be good.

**Review by VSevagen — CHANGES_REQUESTED** [review]
The PR looks good overall. It just needs some slight changes in their respective configs

File: docs/components/scenarios.md (hunk 1 above)
Comment (vchrombie) [reply docs/components/scenarios.md:7]: Done!

**Comment by vchrombie** [comment]
> Also, if we can keep the content within 90 characters per line, it would be good.

Yes, I updated the PR.
For future contributors, there is a VSCode extension to automate this. Please check [Rewrap](https://github.com/stkb/Rewrap). Select the text and `alt + q`.

**Comment by vchrombie** [comment]
@VSevagen could you please check it again?

**Comment by VSevagen** [comment]
> @VSevagen could you please check it again?

Looks good. But right now this is how your component section looks like.
Screenshot_2021-09-27_15-46-44

You need to add an `index.md` file with only the table of content and some lines about about the section. Then change the config of `components.md` to match the other files (workflow.md and scenarios.md)

**Review by VSevagen — APPROVED** [review]

**Comment by vchrombie** [comment]
> You need to add an `index.md` file with only the table of content and some lines about about the section. Then change the config of `components.md` to match the other files (workflow.md and scenarios.md)

I would keep it that way
- /components
- /components/workflow
- /components/scenarios

**Comment by vchrombie** [comment]
Preview: https://vchrombie.github.io/grimoirelab-tutorial/components

**Review by VSevagen — APPROVED** [review]

**Comment by vchrombie** [comment]
Thanks for the review @VSevagen. :+1:

**Pull request merged**